modules/51_internalizeDamages/TCitr/datainput.gms.nash

```diff
@@ -0,0 +1,31 @@
+*** |  (C) 2006-2020 Potsdam Institute for Climate Impact Research (PIK)
```

Comment:
I think this file (.gms.nash) should not be submitted

Reply:
You are right, sorry about that.